### PR TITLE
Feat: Authorino UI basic test

### PR DESCRIPTION
**Pull request by FedeAlonso**
New tests for the Authorino (Kserve Authentication operator) feature.
Those are basic tests, does not include edge cases.
Jira: https://issues.redhat.com/browse/RHOAIENG-4603

### CODE REVIEW

**Comment by asanzgom** [comment]
@FedeAlonso Can you provide a successful run of these Test Cases on the rhods-ci-pr-test job?

**Comment by FedeAlonso** [comment]
> @FedeAlonso Can you provide a successful run of these Test Cases on the rhods-ci-pr-test job? (build number)

@asanzgom Sure 
I've created a run and it was a Success, but It has been already deleted (it was the /rhods-ci-pr-test/3024/ and has finished less than 15 mins ago.)
This an screenshot from the received mail. I can send it personally to you if needed.
image

File: ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_UI.robot [hunk 1]
```diff
@@ -195,6 +195,134 @@ Verify Non Admin Can Serve And Query A Model Using The UI  # robocop: disable
...
     [Teardown]    Run Keywords    Clean Up DSP Page    AND    Close Browser    AND    Switch Browser    1
 
+Verify User Can Serve And Query A Token Protected Model Using The UI
+    [Documentation]    Deploying and querying a Token Protected LLM model
+    ...                using Kserve and Caikit runtime
+    [Tags]    Sanity    Tier1    RHOAIENG-4603
+    [Setup]    Set Up Project    namespace=${TEST_NS}
+    ${test_namespace}=    Set Variable     ${TEST_NS}
+    ${flan_model_name}=    Set Variable    flan-t5-small-caikit
+    Deploy Kserve Model Via UI    ${flan_model_name}    Caikit    kserve-connection    flan-t5-small/${flan_model_name}
```
[the remark this answers is not in the document]
Comment (FedeAlonso) [reply ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_llm_UI.robot]: Done in [5d48ca7](https://github.com/red-hat-data-services/ods-ci/pull/1485/commits/5d48ca7accaecba47d473a5ee3660b4352769d87) 
Thanks @bdattoma :)

**Comment by FedeAlonso** [comment]
Test results after last modifications:
image

**Review by manosnoam — APPROVED** [review]
Thank you Fede for these new Autherino tests!

**Review by bdattoma — APPROVED** [review]
